procurify-dashboard/index.html

```diff
@@ -0,0 +1,317 @@
+                    </div>
+                    </div>
+                    <div class="activity-heading row">
+                    <div class="col">
+                    <div class="user-info activity-status-wrap">
+                      <p class="activity-status">Item Purchased</p>
+                      <p class="sub-info">from <span>Order #113</span></p>
+                    </div>
+                  </div>
+                    </div>
+                  <!-- <div class="activity">
+                    <p class="activity-full">Croissant danish candy canes bear claw dragée pie gingerbread. Dragée jelly beans jelly pudding bonbon. Chocolate apple pie toffee cookie bonbon oat cake halvah. Gummies marzipan muffin candy pastry chocolate bar.</p>
+                  </div> -->
+            </div>
+            <div class="activity-wrap">
+                  <div class="activity-heading user-details row">
+                      <div class="col-6">
+                          <a class="object">
+                            <div class="object-icon">
+                              <img src="http://abduzeedo.com/sites/default/files/originals/63576860118469.5a3c63a44da98.jpg" alt="">
+                            </div>
+                          </a>
+                          <div class="user-info">
+                            <h5>Sohn Jtamos</h5>
+                          </div>
+                        </div>
+                        <div class="user-info activity-order-info col">
+                      <p class="sub-info activity-date">Aug 10, 2018, 12:45 PM</p>
+                    </div>
+                    </div>
+                    <div class="activity-heading row">
+                    <div class="col">
+                    <div class="user-info activity-status-wrap">
+                      <p class="activity-status">Item Purchased</p>
+                      <p class="sub-info">from <span>Order #113</span></p>
+                    </div>
+                  </div>
+                    </div>
+                  <!-- <div class="activity">
+                    <p class="activity-full">Croissant danish candy canes bear claw dragée pie gingerbread. Dragée jelly beans jelly pudding bonbon. Chocolate apple pie toffee cookie bonbon oat cake halvah. Gummies marzipan muffin candy pastry chocolate bar.</p>
+                  </div> -->
+            </div>
+            <div>
+              <div class="budgets-overview-wrap">
+                <div class="budgets-overview">
+                    <div class="panel">
+                        <div class="">
+                          <div class="panel-title">Budgets Overview</div>
+                        </div>
+                        <div class="field">
+                            <label for="">Location</label>
+                            <div class="styled-select">
+                            <select class="budget-dropdown" name="" id="">
```

Comment:
When we start to add functionality to the page with javascript. It will be a good habit to get in to add unique names to form inputs. Remember, no spaces!